medmodels/medrecord/tests/test_overview.py

```diff
@@ -59,6 +59,19 @@ def create_medrecord():
 
```

Comment:
This was no remark before but maybe would be nice to show a test for overview_nodes and overview_edges?